src/main.rs

```diff
@@ -149,20 +159,22 @@ async fn insert_receipts(
             _ => None,
         })
         .flatten()
-        .collect::<Vec<AccessKey>>();
+    );
 
     info!(
         target: INDEXER_FOR_WALLET,
         "Handling receipts related to AccessKey, amount {}",
         access_keys.len()
     );
     if !access_keys.is_empty() {
-        diesel::insert_into(schema::access_keys::table)
+        match diesel::insert_into(schema::access_keys::table)
             .values(access_keys)
             .on_conflict_do_nothing()
             .execute_async(&pool)
-            .await
-            .unwrap();
+            .await {
+            Ok(v) => eprintln!("{:#?}", v),
+            Err(e) => eprintln!("{:#?}", e),
```

Comment:
Retry forever as otherwise we may skip some access keys and never get back to them.